packages/expo-local-authentication/android/build.gradle

```diff
@@ -72,8 +72,8 @@ android {
 
   defaultConfig {
     minSdkVersion safeExtGet("minSdkVersion", 21)
-    targetSdkVersion safeExtGet("targetSdkVersion", 30)
-    versionCode 30
+    targetSdkVersion safeExtGet("targetSdkVersion", 31)
+    versionCode 31
```

Comment:
Updating the `versionCode` is not required — this is updated automatically when we publish the package.

```suggestion
    versionCode 30
```